packer/conf/hooks/environment

```diff
@@ -3,30 +3,25 @@ set -eu
 
 echo '+++ :house_with_garden: Setting up the environment'
 
-[[ -f ~/.docker/config.json ]] && rm ~/.docker/config.json
-[[ -f ~/.dockercfg ]] && rm ~/.dockercfg
+source ~/cfn-env
 
- # handle docker hub authentication
-if [[ -n "${DOCKER_HUB_AUTH:-}" ]] ; then
-  echo "DOCKER_HUB_AUTH is no longer supported, use DOCKER_HUB_USER, DOCKER_HUB_PASSWORD and DOCKER_HUB_EMAIL."
-  exit 1
-elif [[ -n "${DOCKER_HUB_USER:-}" && -n "${DOCKER_HUB_PASSWORD:-}" && -n "${DOCKER_HUB_EMAIL:-}" ]] ; then
-  echo "~~~ Authenticating with :docker: as ${DOCKER_HUB_USER}"
-  docker login --username="${DOCKER_HUB_USER}" --password="${DOCKER_HUB_PASSWORD}" --email="${DOCKER_HUB_EMAIL}"
-elif [[ -n "${DOCKER_HUB_USER:-}" && -n "${DOCKER_HUB_PASSWORD:-}" ]] ; then
-  echo "Missing DOCKER_HUB_EMAIL."
+if [[ -n "$BUILDKITE_SECRETS_BUCKET" ]] && [[ -z "$BUILDKITE_SECRETS_KEY" ]] ; then
+  echo "A SecretsBucket was provided, but no encryption key via BUILDKITE_SECRETS_KEY."
   exit 1
 fi
 
+SSH_KEY_URL="${SSH_KEY_URL:-s3://${BUILDKITE_SECRETS_BUCKET}/id_rsa_github}"
```

Comment:
Is it worth keeping this user-specifiable given it's coupled to `--sse-c-key $BUILDKITE_SECRETS_KEY` etc?
If anything perhaps just the `id_rsa_github` component could be `${SSH_KEY_NAME:-id_rsa_github}`.